client/src/components/app_server_status/AddAppServerForm.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Button, Col, FormControl, Grid, Row } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+import Http from '../../service/Http'
+
+export default class AddAppServerForm extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      enabled: false,
+      name: '',
+      url: ''
+    }
+  }
+
+  render () {
+    return (
+      <Grid>
+        <Row>
+          <Col md={5}>
+            <FormControl
+              type="text"
+              placeholder="Nombre"
+              onChange={this.handleChange('name')}
+              value={this.state.name}
+            />
+          </Col>
+          <Col md={5}>
+            <FormControl
+              type="text"
+              placeholder="URL"
+              onChange={this.handleChange('url')}
```

Comment:
A nadie le importa pero creo que la idea de componentes de React es que tengan cero lógica. Definí el handler dentro de un componente

Reply:
Sip, o sea, mas o menos. Por lo que lei, una forma prolija de hacerlo es separar en component y container. Poner la presentacion en el primero y la logica en el segundo, digo mas o menos porque los dos son componentes react. Ver https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0

Reply:
despues tengo que ver un poco de tutorial/documentación porque no estoy entendiendo bien la idea de qué es un react.component

igual ese blog post me da una idea